modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -293,20 +294,70 @@ def min(self, **kwargs):
         return self._agg("min", **kwargs)
 
     def sum(self, **kwargs):
+        min_count = kwargs.pop("min_count")
+        if min_count != 0:
+            return super().sum(
+                min_count=min_count,
+                **kwargs,
+            )
         return self._agg("sum", **kwargs)
 
     def mean(self, **kwargs):
         return self._agg("mean", **kwargs)
 
+    def nunique(self, axis=0, dropna=True):
+        if axis != 0 or dropna is False:
+            return super().nunique(axis=axis, dropna=dropna)
+        # TODO: check that the OmniSci server is running with the enabled watchdog
+        if "float" in self.dtypes.values:  # and is_watchdog_enabled():
```

Comment:
In this case, OmniSci' watchdog (enabled by default) throws an exception (omnisci/omniscidb#90). Defaulting to pandas here to ensure pandas concordance.